src/views/admin/components/metadata-types/relationship/class-tainacan-relationship.php

```diff
@@ -251,9 +251,13 @@ private function get_item_html($item) {
 			
 			if (is_string($link)) {
 				
-				$return = "<a data-linkto='item' data-id='$id' href='$link'>";
-				$return.= $label;
-				$return .= "</a>";
+				if ( 'publish' === $item->WP_Post->post_status ) {
```

Comment:
Instead of checking directly to the `public` post status, we should get the post status object and verify whether it's public or not.

Something like:

```
$post_status_object = get_post_status_object( $item->get_status() );
if ( $post_status_object->public ) {
```

Theoretically, you can register more post statuses and they can be public or private. It's nice to account for that.

Also, don't we want to check whether the current user has the capability to view the item and, in this case, render the link?